tests/nestml_printer_test.py

```diff
@@ -168,11 +168,11 @@ def test_equations_block_without_comments(self):
         self.assertEqual(block, model_printer.print_node(model))
 
     def test_for_stmt_with_comments(self):
-        stmt = '\n/*pre*/\n' \
-               'for i in 10 - 3.14...10 + 3.14 step -1: # in\n' \
-               'end\n' \
-               '/*post*/\n\n'
-        model = ModelParser.parse_for_stmt(stmt)
+        stmt = '\n  /*pre*/\n' \
+               '  for i in 10 - 3.14...10 + 3.14 step -1: # in\n' \
+               '  end\n' \
+               '  /*post*/\n\n'
+        model = ModelParser.parse_block(stmt)
```

Comment:
Any reason for changing these to `parse_` functions?

Reply:
With newlines no longer going to channel 3, they need to be mentioned explicitly in grammar rules where they may be used. To allow for parsing the newlines in the test cases as they were defined in ``nestml_printer_test.py``, the functions have mostly been replaced by calls to parsing the parent or enclosing grammar rule (e.g. from *assignment* to *block*).